src/config/sites_fixes_v2.json

```diff
@@ -44,6 +44,10 @@
             ]
         },
         {
+            "url": "facebook.github.io/react/*",
+            "selectors":".hero, .nav-main, .nav-home"
```

Comment:
I know what `{common}` does, but it isn't work on react's site. So I tried to fix it like that. But idea about panel for testing is very cool! It will be useful!